help_to_heat/frontdoor/views.py

```diff
@@ -637,14 +638,13 @@ def save_post_data(self, data, session_id, page_name):
 
         uprn = epc_details.get("uprn")
         address = epc_details.get("address")
-        heat_source = epc_details.get("mainheat-description")
 
         epc_data = {
             lmk_field: lmk,
             address_field: address,
             epc_details_field: epc_details,
+            recommendations_field: recommendations,
```

Comment:
save all recommendations rather than say a formatted string, in case we want to change how we show these to energy suppliers in the future